test/functional/specs/Personalization/C28760.js

```diff
@@ -11,6 +11,7 @@ import getResponseBody from "../../helpers/networkLogger/getResponseBody";
 import createResponse from "../../helpers/createResponse";
 import { TEST_PAGE as TEST_PAGE_URL } from "../../helpers/constants/url";
 import createAlloyProxy from "../../helpers/createAlloyProxy";
+import { DECISIONING_PROPOSITION_DISPLAY } from "../../../../src/components/Personalization/constants/eventType";
```

Comment:
Aaron was big on not including constants from src into the tests, and I agree especially for the functional tests. The idea is that you want your tests to make sure everything is working how it should be. If we inadvertently changed the constant, all the tests would still pass. I'm recommending that you revert all the test changes in this PR.